src/main/resources/logback-spring.xml

```diff
@@ -7,10 +7,10 @@
         <encoder>
 			<pattern>${FILE_LOG_PATTERN}</pattern>
 		</encoder>
-		<file>log/alert_audit.log</file>
+		<file>${LOG_PATH:-log}/alert_audit.log</file>
```

Comment:
Where is LOG_PATH defined? Is that a logback variable?

Reply:
LOG_PATH is the environment variable for logging.file.path in the application.properties thats used by spring